tests/integration/end2end/chain_intrablock.yaml

```diff
@@ -43,9 +46,10 @@ pipeline:
           num_embeddings: !@ b1.model.embedder.embedding.num_embeddings
           embedding_dim: 300
         encoder: !PooledRNNEncoder
-          input_size: !g
-            - !@ b2[model][embedder][embedding].embedding_dim
-            - !@ b1.model.embedder.embedding.embedding_dim
+          input_size: !@ b2[model][embedder][embedding].embedding_dim
+          # input_size: !g
```

Comment:
The grid search over links is commented because of memory issues. Locally it is working.